### ISSUE TITLE
Slim import statements for improved packaging performance

### ISSUE THREAD

**Issue by svnhub** [onsetsoftware/automerge-patcher#10]
This is related to import statements in [these files](https://github.com/search?q=repo%3Aonsetsoftware%2Fautomerge-patcher+%22from+%5C%22%40automerge%2Fautomerge%22&type=code)

Recently Automerge added an import entrypoint with a smaller footprint than the default `@automerge/automerge`, namely `@automerge/automerge/slim`

The primary difference between the normal user facing one and the slim one is that the slim one doesn't make any assumptions on the environment that Automerge is loaded in. This makes it easier for package managers running in complex environments to ensure that there are no unintended side-effects of loading your package.

Currently importing automerge-patcher will force an immediate call to Am.init() because that is part of the old import's behaviour.

In our prototype application this breaks when importing in dynamic modules in browser environments with streaming loading of the Automerge WASM backend since the WASM may not have been downloaded yet at the static import resolving phase. Using the slim entrypoint on the other hand leaves it up to the application to determine how and when to launch Automerge.

TLDR: Add /slim to the imports to make things work better in some browser environments

**Comment by acurrieclark** [comment]
Hi @svnhub thanks for reporting this. This makes sense to me. Might you have time for a PR?

If not, I will get this sorted early next week when I am back at work.

**Comment by acurrieclark** [comment]
This should be fixed in [66e21c3](https://github.com/onsetsoftware/automerge-patcher/commit/66e21c359d7165acd559537c7fef41261ce760a1) and released in `v0.14.0`

Closing for now, but please let me know if you are still having trouble.
